src/commands.ts

```diff
@@ -428,6 +431,50 @@ function openInExternalEditor(packageNode: PackageNode) {
     }
 }
 
+/**
+ * Choose DEVELOPER_DIR
+ * @param workspaceContext
+ */
+async function selectXcodeDeveloperDir() {
+    const defaultXcode = await SwiftToolchain.getXcodeDeveloperDir();
+    const selectedXcode = configuration.swiftEnvironmentVariables.DEVELOPER_DIR;
+    const xcodes = await SwiftToolchain.getXcodeInstalls();
+    await withQuickPick(
+        selectedXcode ?? defaultXcode,
+        xcodes.map(xcode => {
+            const developerDir = `${xcode}/Contents/Developer`;
```

Comment:
[nits] the UI and the actual value is a bit different
the list in UI is Xcode paths
but the actual values are the developer paths
it's a little bit different, not sure if engineers would get confused